lib/ansible/module_utils/network/nxos/nxos.py

```diff
@@ -41,6 +43,9 @@
 from ansible.module_utils.six import iteritems, string_types
 from ansible.module_utils.urls import fetch_url
 
+PY2 = sys.version_info[0] == 2
```

Comment:
You can import PY2 and PY3 (and PY34 if you need it) from `module_utils/six`

Reply:
Thanks, updated to use the imports.